source/2019-04-28-the-ember-times-issue-96.md

```diff
@@ -10,91 +10,91 @@ responsive: true
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
-  
+This week:
```

Comment:
I can't add this cleanly in suggestions, can we put them in chronological order?

Reply:
Yes sure, I'll update this in just a bit!